app.py

```diff
@@ -58,5 +60,18 @@ def manual_scrobbling():
     task.start_scrobbling()
     return redirect("/")
 
+@app.route("/test")
+def test_scrobble():
```

Comment:
This should be really part of a unit or integration test ;). Have you written tests in flaks before?